pygmt/helpers/decorators.py

```diff
@@ -327,6 +328,30 @@ def new_module(*args, **kwargs):
     return alias_decorator
 
 
+def insert_alias(module_func):
```

Comment:
Perhaps we should move the code into the `use_alias` decorator, because we only need to "insert aliases into the signature" when aliases are defined by `@use_alias`? It also means we all function support tab completion without any further actions (i.e., `adding @insert_alias`).

Reply:
I agree that combinging `use_alias` and `insert_alias` is the smart move, since I don't see how `insert_alias` would be used on its own.

Reply:
I like the idea of implementing this inside `use_alias` to simplify the diff, but would prefer to not combine by simply moving the code inside `use_alias`. As an alternate solution, my latest commit calls `insert_alias` from within `use_alias`. There are two reasons why I prefer this option to having all the code within `use_alias`. First, I think that it is simpler (i.e., easier to maintain) if individual functions do smaller, isolated tasks. Second, this implementation is compatible with the changes proposed in #1282. Even if that PR doesn't get merged, it seems to me that having `insert_alias` as a separate function could be more compatible with future updates to the decorators (reason 1).